src/ui/public/vislib/lib/axis_config.js

```diff
@@ -117,7 +118,7 @@ export default function AxisConfigFactory() {
     };
 
     isPercentage() {
-      return (this._values.vis._attr.mode === 'percentage');
+      return (this._chartConfig.mode === 'percentage');
```

Comment:
now that you write it down it seems so obvious :) im gonna move this setting to the axis then, and we dont need _chartConfig here.